server/src/handlers.py

```diff
@@ -1,5 +1,43 @@
+import json
+import math
+
 from aiohttp import web
+from schematics.exceptions import DataError
+
+from src import db
+from src.models import room
+from src.utils import parse_json
 
 
 async def ping(request: web.Request) -> web.Response:
     return web.json_response({'msg': 'pong'})
+
+
+async def create_room(request: web.Request) -> web.Response:
+    data = await request.json()
+    some_room = room.Room(data)
+    try:
+        some_room.validate()
+    except DataError as e:
+        return web.json_response(e.to_primitive(), status=400)
+    await db.create_room(data)
+    return web.json_response({'msg': 'OK'}, status=201)
+
+
+async def get_list_rooms(request: web.Request) -> web.Response:
+    page = int(request.query.get('page', '1'))
+    limit = int(request.query.get('limit', '10'))
+    rooms, total_count = await db.get_rooms(page, limit)
+    return web.json_response({
+        "rooms": rooms,
+        "paginator": {
+            "page": page,
+            "limit": limit,
+            "total_pages": math.ceil(total_count / limit),
+        }
+    }, status=200, dumps=parse_json)
+
+
+async def get_room(request: web.Request) -> web.Response:
+    print(request.match_info)
+    return web.json_response({'_id': request.match_info['_id']})
```

Comment:
Seems like the endpoint is not finished yet.
Pls, create issue to implement that endpoint, and remove it from commit